data/account/participationRegistry.go

```diff
@@ -119,10 +124,10 @@ type ParticipationRegistry interface {
 	Delete(id ParticipationID) error
 
 	// Get a participation record.
-	Get(id ParticipationID) (ParticipationRecord, error)
+	Get(id ParticipationID) ParticipationRecord
 
 	// GetAll of the participation records.
-	GetAll() ([]ParticipationRecord, error)
```

Comment:
Most of the changes are related to simplifying the interface call. There isn't a DB error to pass along anymore, so no need for the error value. The caller can check for a zero value / empty list if they need to know that nothing was found.